theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,24 @@
+<nav id="main-navigation">
+  <% function recNav(navig) { %>
+    <ul> 
+      <% for ({title, link, sub_items} of navig) { %>
+        <li>
+            <a href="<%= link %>"><%= title %></a>
+              <% if (sub_items) { %>
+                <a class="has-children" href="<%= link %>"></a>
```

Comment:
Detail/suggestion: Reading the classname, it sounds like `<a>` tag has children, while initially this classname meant to be used on the `<li>` element which technically can have children.